addons/website/static/src/components/dialog/page_properties.xml

```diff
@@ -33,7 +33,7 @@
                 </a>
                 <div t-attf-id="collapseDependencies{{ dependency_index }}" class="collapse" aria-expanded="false">
                     <ul>
-                        <li t-foreach="dependency_value" t-as="item" t-key="item">
+                        <li t-foreach="dependency_value" t-as="item" t-key="item_index">
```

Comment:
Because "dependency_value" is an array, the current item is the current value so "t-key" has to be the index of it.